src/include/btmem.h

```diff
@@ -489,10 +489,23 @@ struct __wt_page_modify {
 	WT_SPINLOCK page_lock;		/* Page's spinlock */
 
 	/*
-	 * The write generation is incremented when a page is modified, a page
-	 * is clean if the write generation is 0.
+	 * The page state is incremented when a page is modified.
+	 *
+	 * WT_PAGE_CLEAN --
+	 *	The page is clean.
+	 * WT_PAGE_DIRTY_FIRST --
+	 *	The page has 1 unreconciled update.
+	 * WT_PAGE_DIRTY --
+	 *	The page has 2 or more unreconciled updates.
```

Comment:
I'd prefer `Two or more updates have been added to the page. `

Reply:
4c3cd34